smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaRecordHelper.java

```diff
@@ -14,6 +16,8 @@
 
 public class KafkaRecordHelper {
 
+    public static final String SMALLRYE_GENERATION_ID = "__smallrye_generationId";
```

Comment:
Does it mean it's SmallRye specific?

Reply:
Yes, to be clear, this should never make it to the user code. It is added on poller thread and removed after.